Makefile

```diff
@@ -171,6 +171,17 @@ generate-proto:  ## Generate Proto Files
 test:  ##
 	pytest -W error --cov src/coherence --cov-report=term --cov-report=html $(UNIT_TESTS) $(E2E_TESTS)
 
+# ----------------------------------------------------------------------------------------------------------------------
+# Run nslookup tests with code coverage
+# ----------------------------------------------------------------------------------------------------------------------
+.PHONY: test-nslookup
+test-nslookup:  ##
+	pytest -W error --cov src/coherence --cov-report=term --cov-report=html \
```

Comment:
Not sure coverage is particularly useful here.  I think we can remove it.

Reply:
ok